bin/config.js

```diff
@@ -64,15 +90,29 @@ const config = (entry, out, bundleName, experimental) => {
   };
 
   // options for minified bundle extend from baseOptions
-  const minOptions = Object.assign({}, baseOptions, {
-    bundleName: `${libraryName}.min.js`,
-    minify: true,
-  });
+  const minOptions = Object.assign(
```

Comment:
Object spread notation is now at Stage-3, I think we can use it to make this (and the next two `Object.assign` calls a bit cleaner.